app/models/manageiq/providers/nuage/manager_mixin.rb

```diff
@@ -35,6 +35,8 @@ def translate_exception(err)
         MiqException::MiqHostError.new("Socket error: #{err.message}")
       when MiqException::MiqInvalidCredentialsError, MiqException::MiqHostError
         err
```

Comment:
@gberginc yes, the problematic error is otherwise returned in the last else statement. And because the error message in the `HTTPBadResponse` is so poorly written, it has very little meaning showing it to the user through a dashboard error notification.